src/tasks/components/NewTaskScheduler/TaskScheduler.scss

```diff
@@ -0,0 +1,46 @@
+.cf-form--footer {
+  position: relative;
+  margin-top: 20%;
+  justify-content: end !important;
```

Comment:
do we need these `!important` in these files? is it possible to get this style using a more specific selector?